shared/src/metabase/mbql/schema.cljc

```diff
@@ -98,15 +98,18 @@
 
 ;; TODO -- currently these are all the same between date/time/datetime
 
-(def ^:private ^{:arglists '([s])} can-parse-date?
+(def ^{:arglists '([s])} can-parse-date?
+  "Can parse an ISO 8601 date (only) or not."
```

Comment:
This is nitpicky but I think this docstring is worded a little bit confusingly. I'm imagining this conversation in The Little Metabaser:

> Q. What does `can-parse-date?` do
>
> A. `can-parse-date?` can parse an ISO 8601 date (only) or not.

😳